src/main/java/org/kcctl/command/GetPluginsCommand.java

```diff
@@ -33,13 +37,19 @@
 import picocli.CommandLine.Command;
 
 @Command(name = "plugins", description = "Displays information about available connector plug-ins")
-public class GetPluginsCommand implements Runnable {
+public class GetPluginsCommand implements Callable<Integer> {
+
+    private final Version requiredVersionForAllPlugins = new Version(3, 2);
+    private final Set<String> defaultPlugins = Set.of("source", "sink");
 
     private final ConfigurationContext context;
 
     @CommandLine.Spec
     CommandLine.Model.CommandSpec spec;
 
+    @CommandLine.Option(names = { "-t", "--types" }, description = "Types of plugins", split = ",")
```

Comment:
Can we make this an [enum option](https://picocli.info/#_enum_types)? That way, we get validation of the passed value(s), and the allowed types will be available for tab completion, too.

Reply:
I've updated the option to use an enum. The completion file has been updated and looks fine but I can't get the completion to work when I type `kcctl get plugins --types=<TAB>` so not sure if it's correct.

Reply:
Interesting, so that does work for me:

```
kcctl get plugins --types
converter         header_converter  predicate         sink              source            transformation
```

Where it falls short is when specifying multiple values, as completion won't work after a comma: `kcctl get plugins --types source,<TAB>`, but I think that's ok.